ui/src/app/workflows/components/submit-workflow-panel.tsx

```diff
@@ -85,6 +89,12 @@ export function SubmitWorkflowPanel(props: Props) {
                         }}
                     />
                 </div>
+                {props.kind === 'ClusterWorkflowTemplate' && (
+                    <div key='namespace' style={{marginBottom: 25}}>
+                        <label>Namespace</label>
+                        <TextInput value={namespace} onChange={setNamespace} />
+                    </div>
+                )}
```

Comment:
This would make more sense at the top of the panel, before the entrypoint, rather than after. Since the first thing you think about is the namespace and then you can get into parameters etc

Reply:
That makes sense. Done in 59dbea3690b1334d8f9b4cfc1e04ef60cc679141. I also updated the screen recording in the PR description